pandas/tests/indexing/test_indexing.py

```diff
@@ -17,6 +17,23 @@
 
 from .test_floats import gen_obj
 
+
+def getitem(x):
```

Comment:
may want to elevate these to pandas/conftest so can use in all indexing tests (in series/frame)